com.krillbite.ThePlan.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: com.krillbite.ThePlan
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: ThePlan.sh
+tags:
+  - proprietary
+finish-args:
+- --share=ipc
+- --socket=x11
+- --device=dri
+- --socket=pulseaudio
+- --persist=.config/unity3d
```

Comment:
Here, actually, it probably respects `$XDG_CONFIG_HOME` env var so can be skipped completely.

Reply:
Unfortunately, Unity 4, which this game is built in, doesn't. It's hard-coded to .config/unity3d and disregards xdg variables.